ui/src/table.test.tsx

```diff
@@ -445,6 +445,82 @@ describe('Table.tsx', () => {
       fireEvent.click(sortCol1)
       expect(getAllByRole('gridcell')[0].textContent).toBe('aa')
     })
+
+    it('Keep sort order when applying filters - no groups', () => {
```

Comment:
Let's add similar test cases for search as well. Do we have a test that checks whether sorting is properly reset on table reset?